tests/perf/smol_str/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! Test that Kani can correctly verify the cedar implementation of `SmolStr`
+//! An ICE was initially reported for this case in:
+//! <https://github.com/model-checking/kani/issues/3312>
+
+#[kani::proof]
+#[kani::unwind(4)]
```

Comment:
Is this annotation required for unwinding to terminate?

Reply:
I don't remember if I just automatically added it. Let me double check

Reply:
I'm asking in case this is an example where Kani cannot automatically unwind. But I don't know if/where we're compiling this information...

Reply:
Thanks!